roles/pulp_user/tasks/main.yml

```diff
@@ -1,24 +1,62 @@
 ---
-
-- name: Get user list
+- name: Get information for each user
   uri:
-    url: "{{ pulp_user_url }}"
+    url: "{{ pulp_user_url }}?username={{ item.username }}"
     user: "{{ pulp_username }}"
     password: "{{ pulp_password }}"
     method: GET
     status_code: 200
     force_basic_auth: true
-  register: users_list_result
+  loop: "{{ pulp_users }}"
```

Comment:
Can we use Jack's trick to hide the passwords here? https://github.com/stackhpc/ansible-collection-pulp/pull/53/files

Reply:
Could be a separate change

Reply:
Good idea - I'll add as a separate change